app/screens/Lists/BooksList/components/BookItem/index.js

```diff
@@ -1,12 +1,32 @@
 import React, { Component } from 'react';
-import { View, Image, StyleSheet, Text } from 'react-native';
+import { View, Image, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { StackActions } from 'react-navigation';
 
 import styles from './styles';
 import placeholder from './assets/book.png';
 
-function BookItem({ image, title, author }) {
-  return (
-    <View style={styles.itemContainer}>
+class BookItem extends Component {
+  handlePressItem = () => {    
```

Comment:
you must declare the handlePress function on the BookList index. So this function should receive the id and the title.

`handlePressItem(id, title) = ....`   <-- of BookList index file

and this handle just should pass the specific params of the item to the father function.